core/dbt/graph/selector.py

```diff
@@ -103,11 +94,11 @@ def get_nodes_from_criteria(
         try:
             collected = self.select_included(nodes, spec)
         except InvalidSelectorException:
-            valid_selectors = ", ".join(self.SELECTOR_METHODS)
```

Comment:
Is there a reason to move this into `SelectorReportInvalidSelector`?

Reply:
I'm trying to take the approach where the event represents all the necessary data, and the responsibility to turn data into human-readable strings is entirely up to the event methods. `valid_selectors` looks like it's entirely about turning data into strings so I moved it.

Reply:
Makes sense.  Just want to make sure I'm consistent.